src/patternfly/base/tokens/_tokens-font.scss

```
@@ -62,4 +62,33 @@
   --pf-t--global--border--width--control--default: var(--pf-t--global--border--width--100);
   --pf-t--global--border--width--control--hover: var(--pf-t--global--border--width--100);
   --pf-t--global--border--width--control--active: var(--pf-t--global--border--width--200);
+
+  // box-shadow
+  --pf-t--global--box-shadow--sm: 0px, 1px, 2px, 0px rgba(0, 0, 0, 0,0,0, 16%);
+  --pf-t--global--box-shadow--sm--top: 0px, -1px, 2px, 0px rgba(0, 0, 0, 16%);
```

This may be known since you're still working on these, but we need to remove the commas between the values and it's probably a good time to go ahead and switch to `rgb()` since it supports the alpha channel and `rgba()` is deprecated (though it still works fine). 

```suggestion
  --pf-t--global--box-shadow--sm--top: 0px -1px 2px 0px rgb(0 0 0 / 16%);
```